src/features/PersonList/popularPeopleSaga.js

```diff
@@ -3,39 +3,40 @@ import { getPopularPeople } from "../../api/getPopularPeople";
 import {
   fetchPopularPeopleSuccess,
   fetchPopularPeopleError,
-  fetchCurrentPeoplePage,
+  fetchPopularPeoplePageNumberForApi,
 } from "./popularPeopleSlice";
 
 function* fetchPopularPeopleHandler({ payload: pageNumber }) {
   try {
-    const firstPageNumberForApi = pageNumber + (pageNumber - 1) * 2;
-    const secondPageNumberForApi = firstPageNumberForApi + 1;
-    const thirdPageNumberForApi = firstPageNumberForApi + 2;
+    const firstPageFromApi = yield call(getPopularPeople, pageNumber);
 
-    const popularPeopleFirstPage = yield call(
-      getPopularPeople,
-      firstPageNumberForApi
-    );
-    const popularPeopleSecondPage = yield call(
-      getPopularPeople,
-      secondPageNumberForApi
-    );
-    const popularPeopleThirdPage = yield call(
-      getPopularPeople,
-      thirdPageNumberForApi
-    );
+    const secondPageFromApi = yield call(getPopularPeople, pageNumber + 1);
```

Comment:
Great, now it is better for reading of code!

Reply:
I think so too :)